packages/server/graphql/public/mutations/addReactjiToReactable.ts

```diff
@@ -0,0 +1,221 @@
+  ] as ValueOf<typeof pgDataloaderLookup> | null
+  const rethinkDbTable = rethinkTableLookup[reactableType as keyof typeof rethinkTableLookup]
+  if (pgLoaderName) {
+    reactable = await dataLoader.get(pgLoaderName).loadNonNull(reactableId)
+  } else {
+    reactable = (await r.table(rethinkDbTable).get(reactableId).run()) as Reactable
+  }
+  if (!reactable) {
+    return {error: {message: `Item does not exist`}}
+  }
+  const {reactjis} = reactable
+  const verifiedType = getReactableType(reactable)
+  if (verifiedType !== reactableType) {
+    return {error: {message: `Unknown item`}}
+  }
+  const meetingMemberId = toTeamMemberId(meetingId, viewerId)
+  const viewerMeetingMember = await dataLoader.get('meetingMembers').load(meetingMemberId)
+  if (!viewerMeetingMember) {
+    return {error: {message: `Not a member of the meeting`}}
+  }
+
+  // VALIDATION
+  if (!emojiIds.includes(reactji)) {
+    return {error: {message: `invalid emoji`}}
+  }
+
+  if (!isRemove) {
+    const groupedReactjis = getGroupedReactjis(reactjis, viewerId, reactableId)
+    const nextReactjiId = `${reactableId}:${reactji}`
+    const isReactjiPresent = !!groupedReactjis.find((agg) => agg.id === nextReactjiId)
+    if (!isReactjiPresent && groupedReactjis.length >= Threshold.MAX_REACTJIS) {
+      return {error: {message: `Reactji limit reached`}}
+    }
+  }
+
+  // RESOLUTION
+  const subDoc = {id: reactji, userId: viewerId}
+  if (pgLoaderName) {
+    const numberReactableId = TeamPromptResponseId.split(reactableId)
+    if (isRemove) {
+      await removeTeamResponseReactji.run(
+        {id: numberReactableId, reactji: {shortname: reactji, userid: viewerId}},
+        getPg()
+      )
+    } else {
+      await appendTeamResponseReactji.run(
+        {id: numberReactableId, reactji: {shortname: reactji, userid: viewerId}},
+        getPg()
+      )
+    }
+
+    dataLoader.get(pgLoaderName).clear(reactableId)
+  } else {
+    if (isRemove) {
+      await r
+        .table(rethinkDbTable)
+        .get(reactableId)
+        .update((row: RDatum<Comment | Reflection>) => ({
+          reactjis: row('reactjis').difference([subDoc]),
+          updatedAt: now
+        }))
+        .run()
+    } else {
+      await r
+        .table(rethinkDbTable)
+        .get(reactableId)
+        .update((row: RDatum<Comment | Reflection>) => ({
+          reactjis: r.branch(
+            row('reactjis').contains(subDoc),
+            row('reactjis'),
+            // don't use distinct, it sorts the fields
+            row('reactjis').append(subDoc)
+          ),
+          updatedAt: now
+        }))
+        .run()
+    }
+  }
+
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const {meetingType, teamId} = meeting
+  const team = await dataLoader.get('teams').loadNonNull(teamId)
+
+  const reactableCreatorId = getReactableCreatorId(reactableType, reactable, meeting)
+
+  let addedKudosId = null
+  if (
```

Comment:
Removed feature flag check for v1, but keep feature flag added in DB for possible future versions